Simplenote/Note+Links.swift

```diff
@@ -36,4 +36,9 @@ extension Note {
 
         return SimplenoteConstants.simplenotePublishedBaseURL + targetURL
     }
+
+    static func instancesOfReferenceToNoteWith(simperiumKey: String, in content: String) -> Int {
+        let internalLink = SimplenoteConstants.simplenoteScheme + "://" + SimplenoteConstants.simplenoteInterlinkHost + "/" + simperiumKey
+        return content.occurancesOf(internalLink)
+    }
```

Comment:
How about `func instancesOfReference(to note: Note) -> Int`? Seems cleaner and we can put all the logic here.
We can also reuse `plainInternalLink` here, which will remove code duplication.

Reply:
Thanks for the suggestions on how I can make that work with the unit tests.  Was able to get that working with `func instancesOfReference(to note: Note) -> Int`